newrelic-nri-statsd.yaml

```diff
@@ -32,6 +32,14 @@ pipeline:
       install -Dm755 nri-statsd.sh "${{targets.destdir}}"/home/nonroot/nri-statsd.sh
       install -Dm755 run-statsd.sh "${{targets.destdir}}"/home/nonroot/run-statsd.sh
 
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Make the nri-statsd.sh script compatible with Dockerfile entrypoint
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /home/nonroot/nri-statsd.sh "${{targets.subpkgdir}}"/nri-statsd.sh
```

Comment:
Can this be avoided by explicitly setting the entry point as `/home/nonroot/nri-statsd.sh`?

Reply:
its working when I set `./nri-statsd.sh` not when its `/nri-statsd.sh` 🤷🏻‍♂️

Reply:
but good suggestion, let me try that one!

Reply:
it sounds like its worked! 

```
(っ◔◡◔)っ  docker container run --pull always --rm -p 8125:8125 ghcr.io/developer-guy/newrelic-nri-statsd:latest                                             1.22.2
WARNING: configuration file not found, generating one based on default values.
make sure you have provided the required NR_API_KEY and optionally
NR_ACCOUNT_ID if using Insights events, via environment variables.
time="2024-04-07T14:09:31Z" level=info msg="Starting server"
time="2024-04-07T14:09:31Z" level=info msg="No cloud provider specified"
time="2024-04-07T14:09:31Z" level=info msg="internal metrics OFF, to enable set 'statser-type' to 'logging' or 'internal'" backend=newrelic
time="2024-04-07T14:09:31Z" level=fatal msg="could not init backend \"newrelic\": [newrelic] api-key is required to flush to insights & metrics backends"
```


```
(っ◔◡◔)っ  crane config --platform linux/arm64 ghcr.io/developer-guy/newrelic-nri-statsd:latest | jq '.config' 
{
  "Entrypoint": [
    "/home/nonroot/nri-statsd.sh"
  ],
  "Env": [
    "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
    "SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt"
  ],
  "User": "65532",
  "WorkingDir": "/home/nonroot"
}
```

Reply:
thanks @EyeCantCU !!!